server/src/main/java/com/lunchtime/controllers/RestaurantController.java

```diff
@@ -35,6 +36,13 @@ public RestaurantController(RestaurantService restaurantService) {
             return ResponseEntity.badRequest()
                 .build();
         }
+        Optional<Person> res = personRepository.findById(restaurant.getPersonId());
+        if (res.isPresent()) {
+            Person person = res.get();
+            person.setRoleId((long) 2);
```

Comment:
We can use simple `2L` here.

Reply:
2l what ? fore what ?

Reply:
I mean 2L - 2 Long

Reply:
what's the difference?

Reply:
It is casted here int 2 to long primitive type. We can do that without casting just writing 2L.

Reply:
i wonder what is the difference between using this or that option if it is significant then i will change it in the code if there is no difference then it is not a priority at the moment

Reply:
Absolutely :)

Reply:
@Melnyk9506 It makes code more readable and clear. In your manual type casting too many parentheses. You should use `2L`